### ISSUE TITLE
Can I use it without using a docker?

### ISSUE THREAD

**Issue by fahadansilt** [trizin/Telegram-Airdrop-Bot#7]

**Comment by trizin** [comment]
Yes, you will need to install Mongo DB on your machine and change the environment variables.

https://github.com/trizin/Telegram-Airdrop-Bot/blob/e4296bd90b39e17c7225a684834d651d3d20a476/.env.example#L19-L22

**Comment by elusiee** [comment]
Can I use live mongodb instead of localhost?
